packs/classfeatures/sparkling-targe.json

```diff
@@ -44,6 +44,14 @@
                     {
                         "or": [
                             "magical",
+                            "arcane",
+                            "divine",
+                            "occult",
+                            "primal",
+                            "item:trait:arcane",
+                            "item:trait:divine",
+                            "item:trait:occult",
+                            "item:trait:primal",
```

Comment:
I'm not big on including both of these. We should probably be moving to only the `item:trait:x` ones.